tests/phpunit/class-wc-stripe-test.php

```
@@ -22,6 +22,40 @@ public function test_constants_defined() {
+			);
+			add_action( 'woocommerce_review_order_before_submit', [ $config, $method ] );
+		}
+
+		// No TypeError when the hook fires means the shim held.
+		do_action( 'woocommerce_review_order_before_submit' );
+		$this->assertTrue( true );
+
+		foreach ( $legacy_methods as $method ) {
+			remove_action( 'woocommerce_review_order_before_submit', [ $config, $method ] );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_payment_request_configuration_legacy_callbacks_do_not_fatal is not named in camelCase. (undefined)

(CamelCaseMethodName)